src/main/java/org/eclipse/uprotocol/cloudevent/factory/CloudEventFactory.java

```diff
@@ -176,8 +176,10 @@ static CloudEventBuilder buildBaseCloudEvent(String id, String source,
         final CloudEventBuilder cloudEventBuilder = CloudEventBuilder.v1()
                 .withId(id)
                 .withSource(URI.create(source))
+                /* Not needed:
                 .withDataContentType(PROTOBUF_CONTENT_TYPE)
                 .withDataSchema(URI.create(protoPayloadSchema))
```

Comment:
Can we remove the following argument from the method:
Line 174: String protoPayloadSchema,

Reply:
Removal of the argument will cause IoP issues so I prefer not to for now.